js/views/userPageVw.js

```diff
@@ -812,6 +839,17 @@ module.exports = baseVw.extend({
     }
   },
 
+  renderReviews: function (model) {
+    model = model || [];
+    this.reviewsVw = new reviewsView({
```

Comment:
renderReviews() is called from fetchReviews(), which is called from render(). I'm not sure if we actually are re-rendering this view, but it is an extremely common thing to re-render a view as data changes (sometimes many, many times). If we're creating a new reviewsVw and not removing the previous one, we are leaving it as a 'ghost' view and are not giving it a chance to do it's cleanup. E.g. it may (theoretically, don't know if it actually does) bind some event handler to the document, that it removes in remove().

registerChild() will automatically remove any child views when the parent view (this view) is removed. But, in situations were it's appropriate to remove a child view while the parent is still active, it is up to the parent to remove the child.

If renderFollowing() is not doing so, it is an oversight on our part and should.